### DISCUSSION TITLE
How did you make that homepage?

### DISCUSSION THREAD

**Discussion by zina0316** [KosmosisDire/obsidian-webpage-export#516] [General]
I recently came across the [homepage](https://docs.obsidianweb.net/) of this project
and that is so cool!
Is that made with a plugin or something?

**Comment by zina0316** [comment]
shoot I searched it with the keyword "homepage" so I missed [this](https://github.com/KosmosisDire/obsidian-webpage-export/discussions/379) one

**Comment by KosmosisDire** [comment]
Haha no worries!
